src/utils/history/recordHistory.ts

```diff
@@ -348,7 +348,7 @@ export class RecordHistory {
     ).data;
     let difference: any;
     if (versions.length === 0) {
-      difference = this.getDifference(null, filteredData);
+      difference = await this.getDifference(null, filteredData);
```

Comment:
@ketanreliefapps I had to close this one
it does not seem to bring anything new
I believe it was fixed by another ticket without noticing it